frontend/src/metabase/public/components/EmbedModal/StaticEmbedSetupPane/StaticEmbedSetupPane.tsx

```diff
@@ -107,19 +113,17 @@ export const StaticEmbedSetupPane = ({
     embeddingParams,
   });
 
-  const iframeUrl = useMemo(
+  const iframeUrlWithoutHash = useMemo(
```

Comment:
I need the URL to stay consistent when only the appearance option is changed. Otherwise, it will cause the iframe to rerender.

Reply:
Isn't this only used in `iframeUrl` which also uses the hash params?

Reply:
yeah, this variable is used down below `const iframeUrl = iframeUrlWithoutHash + optionsToHashParams(displayOptions);`

Basically, I need to base URL to stay consistent, so I separate the calculation of base URL and hash. So if only the hash changes, the base URL would stay the same.